lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -669,7 +715,7 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
        }) do
     Enum.concat(
       primitives,
-      Enum.flat_map(clones, &Map.get(&1, :primitives, [])) ++
+      Enum.flat_map(clones, &(&1 |> Map.get(:primitive, []) |> List.wrap())) ++
```

Comment:
This was a small bug we had. We didn't notice as we were not using cloned resources coming from this function